resources/basics.py

```diff
@@ -469,10 +469,25 @@ def methylation_sites_mt_path(hail_version=CURRENT_HAIL_VERSION, ref='GRCh37'):
 genome_evaluation_intervals_path_hg38 = "gs://gnomad-public/intervals/hg38-v0-wgs_evaluation_regions.hg38.interval_list"
 # More can be found at gs://broad-references/hg19
 
-vep_config = 'gs://hail-common/vep/vep/vep85-loftee-gcloud.json'
 
-# Annotations
-context_mt_path = 'gs://gnomad-resources/context/hail-0.2/context_processed.mt'
+def vep_config_path(reference='GRCh37'):
+    if reference not in REFERENCES:
+        return DataException("Select reference as one of: {}".format(','.join(REFERENCES)))
+    if reference == 'GRCh37':
+        return 'gs://hail-common/vep/vep/vep85-loftee-gcloud.json'
+    elif reference == 'GRCh38':
+        return 'gs://hail-common/vep/vep/vep95-GRCh38-loftee-gcloud.json'
+
+vep_config = vep_config_path()
+
+
+def context_ht_path(reference='GRCh37'):
+    if reference not in ('GRCh37', ):
+        return DataException("Reference must be GRCh37")
+    if reference == 'GRCh37':
+        return 'gs://gnomad-public/papers/2019-flagship-lof/v1.0/context/Homo_sapiens_assembly19.fasta.snps_only.vep_20181129.ht'
```

Comment:
REFERENCE can be a a dict of dicts for multiple values. I think in terms of configs it's better to put all config related things in one place so it can easily be changed.

Reply:
I generally agree - the reason we didn't do this before is we wanted the flexibility of functions to specify things like hail version. But I think that ship has sailed now, so unless anyone has any objections, I'll fix it to do so.

Reply:
Hmm, how would the caller specify the hail version? Also, I'm not too familiar with this package but I assume it's okay to just pin it to hail v02 since we'd never import hail v01 and v02, right?

Reply:
Yeah, a bunch of these functions had `hail_version=...` as arguments, but I think the utility of these is long gone